docs/src/main/asciidoc/_galleon/Galleon_provisioning.adoc

```diff
@@ -0,0 +1,158 @@
+[[Galleon_Provisioning]]
+= Provisioning WildFly with Galleon
+
+With the WildFly server being internally built using the https://github.com/wildfly/galleon[Galleon project], WildFly benefits +
```

Comment:
I don't know if we should jump more into the benefits of using Galleon for provisioning rather than our internal use?  Maybe jump straight into something declaring it's benefit over a zip installation?

Reply:
I kept the sentence as it is. It says that because we have refactored WildFly to use galleon, we can benefit from it.

Reply:
+1 @darranl

Reply:
Ok, I have simplified to state the advantages.